frontend/src/components/v2/Alert/Alert.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from "react";
+import { type VariantProps, cva } from "cva";
+
+import { cn } from "@app/helpers/classNames";
```

Comment:
Its better just use twMerge itself here. I don't see the value in reusing like this. We are planning to reduce as much as file. So please keep the other component style